### ISSUE TITLE
No CHANGELOG provided for multiple releases recently

### ISSUE THREAD

**Issue by maiconcarraro** [radix-ui/primitives#3292]
Similar situation from few months ago: https://github.com/radix-ui/primitives/issues/2962

![image](https://github.com/user-attachments/assets/ce881c4f-7277-48ee-aca9-6baaf76c625e)

Checked on https://www.radix-ui.com/primitives/docs/overview/releases and there is no information yet, just opening this issue to keep track it's missing. Appreciate the hard work!

**Comment by chaance** [comment]
Yeah sorry, just haven't had time to update the site. Only thing in the most recent release was adding support for callback refs to return cleanup functions (new feature in React 19). Cutting another shortly to relax a dependency that will eliminate some compat warnings when installing. I'll try to update the site this week!

**Comment by maiconcarraro** [comment]
Latest changelogs are available, closing this, thank you! ✨
